### ISSUE TITLE
The "pull-request-title-pattern" option is not respected

### ISSUE THREAD

**Issue by kettanaito** [google-github-actions/release-please-action#436]
### TL;DR

The custom value for the `pull-request-title-pattern` option in the workflow has no effect: the default pull request title is being used instead.

### Expected behavior

The action uses the pattern provided as the value of the `pull-request-title-pattern` option in the workflow file.

### Observed behavior

The default pull request title pattern is used anyway.

### Action YAML

```yaml
name: CI
on:
  push:
    branches: [main]

jobs:
  build-and-test:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        id: release
        with:
          command: manifest-pr
          pull-request-title-pattern: "chore(${component}): release ${version}"
```

### Log output

```text
No log output.
```

### Additional information

I've also tried setting the `group-pull-request-title-pattern` option in the `release-please-config.json`, which resulted in no change: the default pattern is always used.

It's worth mentioning I'm integrating this in a monorepo, maybe it has something to do with the issue. The release config is still in the root of the repo, just like GitHub workflows.

**Comment by joeyparrish** [comment]
I had the same issue with my monorepo, and just discovered that `pull-request-title-pattern` in the config file (at the root) is being used instead.  The action's input is ignored when there's a manifest.

**Comment by joeyparrish** [comment]
No, wait.  It looked like the case while debugging in the action's source, but it's not actually working there, either.

**Comment by joeyparrish** [comment]
Interestingly, with `"separate-pull-requests": false`, I see that `group-pull-request-title-pattern` is being used from the config file.  But with `"separate-pull-requests": true`, I see that none of the three pattern settings are used (group, regular, action input).
